lib/tasks/foreman_templates_tasks.rake

```diff
@@ -24,6 +27,23 @@ namespace :templates do
     puts results.join("\n")
   end
 
+  task :import => :sync
```

Comment:
Nitpick: Shouldn't this the other way around? :sync is deprecated, so the code should now be at :import